peps/pep-0796.rst

```diff
@@ -0,0 +1,322 @@
+paths.
+
+A proof-of-concept of this is implemented in the author's branch,
+`rickeylev/feat.relative.pyvenv.home <https://github.com/python/cpython/compare/main...rickeylev:cpython:feat.relative.pyvenv.home>`__.
+
+Backwards Compatibility
+=======================
+
+Tools that work around the absolute ``home`` key limitation the way Bazel
+and venvstacks currently do (omitting the ``home`` key, or editing it after
+moving the environment) will be unaffected.
+
+While the PEP author and sponsor aren't aware of any projects that work around
+the limitation by carefully controlling the current working directory used to
+launch the deployed Python environments on target systems, any such projects
+would be unaffected if they already ensured the working directory was set to
+the folder containing ``pyvenv.cfg`` (which seems like a plausible choice,
+since that is typically the root directory of the virtual environment). In the
+even more unlikely case where that assumption doesn't hold, tools generating
+relative virtual environment paths will typically be aware of the underlying
+base runtime Python version, and hence able to update the emitted relative path
+accordingly.
+
+Security Implications
+=====================
+
+A relative path in :file:`pyvenv.cfg` may resolve differently depending on the
+location of the virtual environment. This *could* point to a surprising,
+potentially malicious, location.
+
+However, this risk already exists today because a relative path isn't
+_rejected_, but resolved relative to the current working directory. This PEP
+just changes the anchor point to ``pyvenv.cfg`` itself.
+
+Similarly, the same concern exists for absolute paths. The two are
+fundamentally the same because they both rely on trusting whoever created
+the ``pyvenv.cfg`` file, which requires having run another tool or downloaded
+something from elsewhere.
+
+
+How to Teach This
+=================
+
+Teaching this should be straightforward: if you use a relative path in
+``pyvenv.cfg``, then it's relative to the directory containing the
+``pyvenv.cfg`` file. This is simple to explain and easy to understand for
+anyone that is already familiar with handling relative filesystem paths.
+
+
+Reference Implementation
+========================
+
+A reference implementation is available by using the combination of:
+
+* Python runtime from `rickeylev/feat.relative.pyvenv.home <https://github.com/python/cpython/compare/main...rickeylev:cpython:feat.relative.pyvenv.home>`__
+* Relative venv from `rickeylev/relvenv <https://github.com/rickeylev/relvenv>`__
+
+And following the
+`relvenv README <https://github.com/rickeylev/relvenv/blob/main/README.md>`__.
+
+Open Issues
+===========
+
+This PEP does not specify how to create a ``pyvenv.cfg`` with a relative path,
+nor how downstream tools (e.g. installers) should identify them or process
+them. These questions are best addressed separately by tool owners.
```

Comment:
This isn't an open issue, but rather than intentional omission. Perhaps just include a variant of this paragraph in the rationale subsection about not fully specifying portable virtual environments?